src/containers/Login/Password.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from "react";
+import EmailConfirmation from "./EmailConfirmation";
+import UpdatePassword from "./UpdatePassword";
+
+class Password extends Component {
```

Comment:
@Humnaa rename this component and file to "ForgotPassword"

Reply:
renamed